pkg/testing/integration/program.go

```diff
@@ -740,9 +741,7 @@ func GetLogs(
 
 	// Validate logs from example
 	logs, err := ops.GetLogs(query)
-	if !assert.NoError(t, err) {
-		return nil
-	}
```

Comment:
Guessing this behavioural change has no practical effect?

Reply:
Yup, same as above, this change in behaviour is intended (and no change when the test succeeds)